Code/autopkglib/URLDownloader.py

```diff
@@ -125,7 +126,8 @@ def main(self):
         # Download URL.
         url_handle = None
         try:
-            request = urllib2.Request(url=self.env["url"])
+            url = urllib.quote(self.env["url"], 'http://')
```

Comment:
The second parameter to urllib.quote is a list of "safe" characters; that is, a list of characters to NOT encode. Not sure what you are aiming for here...